### DISCUSSION TITLE
How can I trust this browser to not hack any accounts?

### DISCUSSION THREAD

**Discussion by DefaultUser9148** [minbrowser/min#1972] [Q&A]
I don't trust this browser to sign in to any account on any website, how can I trust this so that it will not give my email and password to someone or hack any of my accounts. I know this seems stupid, but it happened with another browser like this before. Thanks!

**Comment by PalmerAL** [comment]
I understand your concern; using a browser does require a lot of trust in it. While I can tell you that Min isn't malicious, that isn't very good proof :)

I will say that:
1. Min has been around for a while now (over 6 years). If there was any malicious behavior, there's been a long opportunity for people to find it. 
2. The code is all open-source, so it's possible to inspect it and verify that it's not collecting any of your personal information. You can then compile it yourself to ensure that the code you're running matches the code you've inspected: https://github.com/minbrowser/min#building-binaries.

The other option is to look at the network requests Min is making using something like [Wireshark](https://www.wireshark.org/).

**Comment by DefaultUser9148** [comment]
While don't know anything about coding or that stuff, i definetly didn't
know that the Min browser was released 6 years ago, I will still check out
if it has any suspicious activity, Thank You!
